.mergify.yml

```diff
@@ -1,14 +1,4 @@
 pull_request_rules:
-  - name: Auto merge
-    conditions:
-      - "#changes-requested-reviews-by=0"
-      - label=st:test-and-merge
-      - status-success=Jenkins
-      - status-success=pretest  # github workflows
```

Comment:
The requirements will be managed by Settings -> Branches -> Branch protection rules (master or v9) -> Status checks that are required.